new_platforms/scripts/VirtualMachineSgxSettings.psm1

```diff
@@ -0,0 +1,283 @@
+﻿Import-Module Hyper-V
+
+### -----------------------------------
+### Constants
+### -----------------------------------
+
+$VM_MEMORY_SETTINGS_GUID="4764334d-e001-4176-82ee-5594ec9b530e"
+
+Set-StrictMode -Version 5
+
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+
+### -----------------------------------
+### Strings
+### -----------------------------------
+
+Data Strings
+{
+# culture="en-US"
+ConvertFrom-StringData @'
+    ERR_NO_MEM_SETTING_DATA_OBJECT = Could not retreive the Memory Setting Data object for VM '{0}' from WMI.
+    ERR_NO_VMM = Could not retreive the instance of the Virtual System Management Service from WMI.
+    ERR_VM_STATE = "The VM '{0}' must be Off before its SGX settings may be modified."
```

Comment:
Why are double quotes needed on this line but not on line 22, 27, etc?

Reply:
They are not; removed.